docs/site/migration/auth/oauth2.md

```diff
@@ -6,12 +6,23 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-auth-oauth2.html
 ---
 
-{% include tip.html content="
-Missing instructions for your LoopBack 3 use case? Please report a [Migration docs issue](https://github.com/strongloop/loopback-next/issues/new?labels=question,Migration,Docs&template=Migration_docs.md) on GitHub to let us know.
-" %}
+[`loopback-component-oauth2`](https://github.com/strongloop/loopback-component-oauth2)
+is used to setup a LoopBack application as an [OAuth 2.0](https://oauth.net/2/)
+server. It can be applied when people implement their LoopBack application as an
+OAuth 2.0 provider. For example, the API gateway application.
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3959](https://github.com/strongloop/loopback-next/issues/3959)
-" %}
+At this stage, authentication and authorization in LoopBack 4 focus on the
+communication to third-party servers like
+[google](https://developers.google.com/identity/protocols/oauth2) or
+[facebook](https://developers.facebook.com/docs/facebook-login/), instead of
+turning itself into an OAuth 2.0 provider. And given the complexity of
+[`loopback-component-oauth2`](https://github.com/strongloop/loopback-component-oauth2),
+we decide to postpone the migration guide to next stage.
+
+We have a simplified OAuth 2.0 server implementation in example
+[passport-login](https://github.com/strongloop/loopback-next/tree/master/examples/passport-login),
+which sets up an express application as an OAuth 2.0 provider. The provider is
```

Comment:
> The provider is created in file mock-oauth2-social-app.ts

Do you mean we `also` have another example of an oauth provider in mock-oauth2-social-app.ts ?

The way it is currently worded, it makes it sound as if mock-oauth2-social-app.ts is being used in examples/passport-login .

Reply:
> mock-oauth2-social-app.ts is being used in examples/passport-login 

 @emonddr It is a fixture in the example's test folder. Hmm, is the meaning clear now?